src/confluent_kafka/admin/_group.py

```diff
@@ -94,15 +94,18 @@ class MemberDescription:
         The host where the group member is running.
     assignment: MemberAssignment
         The assignment of the group member
+    target_assignment: MemberAssignment
+        The target assignment of the group member
     group_instance_id : str
         The instance id of the group member.
     """
 
-    def __init__(self, member_id, client_id, host, assignment, group_instance_id=None):
+    def __init__(self, member_id, client_id, host, assignment, target_assignment, group_instance_id=None):
```

Comment:
Don't change the API contract.

```suggestion
    def __init__(self, member_id, client_id, host, assignment, group_instance_id=None, target_assignment=None):
```